src/components/Textarea.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react'
+import {
+  Textarea as RebassTextarea,
+  TextareaProps,
+} from '@rebass/forms/styled-components'
+import ErrorMessage from './ErrorMessage'
+
+export interface ITextarea extends TextareaProps {
+  error?: string
+  fontSize?: number
+  rows?: number
+}
+
+const Textarea: FC<ITextarea> = (props) => {
+  const { error, fontSize, rows, ...rest } = props
+
+  return (
+    <div>
+      <RebassTextarea
+        {...rest}
+        rows={rows}
```

Comment:
Why do you need to explicitly extract the `row` props and not just pass it as part of `...rest`?

Reply:
Okay, I think yes, now it makes sense because we've separated out the Input and Textarea